pkg/cli/interface.go

```diff
@@ -17,14 +17,20 @@ package cli
 import (
 	"github.com/jonboulle/clockwork"
 	"github.com/tektoncd/pipeline/pkg/client/clientset/versioned"
+	k8s "k8s.io/client-go/kubernetes"
 )
 
+type Clients struct {
+	Tekton versioned.Interface
+	Kube   k8s.Interface
+}
+
 // Params interface provides
 type Params interface {
-	// SetKubeConfigPath uses the kubeconfig path to instantiate clientset
+	// SetKubeConfigPath uses the kubeconfig path to instantiate tekton
 	// returned by Clientset function
 	SetKubeConfigPath(string)
-	Clientset() (versioned.Interface, error)
```

Comment:
I wonder if this change could have gone into a different PR? :thinking:

Reply:
That's fine here :wink: